Library/Homebrew/cmd/fetch.rb

```diff
@@ -32,6 +32,11 @@ def fetch
   def fetch_bottle? f
     return true if ARGV.force_bottle? && f.bottle
     return false unless f.bottle && f.pour_bottle?
+    if formula.file_modified?
+      ohai "#{formula.full_name}.rb is modified; fetching source"
```

Comment:
On narrow tmux/screen panes the path will get cut off, but I think that's fine:
![screenshot 2015-07-11 10 25 18](https://cloud.githubusercontent.com/assets/985416/8634688/b62f0be4-27b7-11e5-9f98-d5b3a658a782.png)